### DISCUSSION TITLE
Screenshots of what the site looks like

### DISCUSSION THREAD

**Discussion by gmenziesint** [almenscorner/intunecd-monitor#46] [General]
Hey,

I was wondering if you had any screenshots of what the site looks like? I was planning to set this up and wanted to make sure it was suitable for our needs before I go through the process!

Thanks.

**Comment by almenscorner** [comment]
Sure, here you go

<img width="1855" alt="Screenshot 2024-10-25 at 13 57 28" src="https://github.com/user-attachments/assets/a4daf13d-2888-47e0-96e2-4693ec8adb41">
<img width="1855" alt="Screenshot 2024-10-25 at 14 03 00" src="https://github.com/user-attachments/assets/456dd6a9-e325-4860-bb34-cda482ebf797">
<img width="1855" alt="Screenshot 2024-10-25 at 13 58 13" src="https://github.com/user-attachments/assets/3119e8ea-73da-4498-bb0d-2133f184dc45">
<img width="1855" alt="Screenshot 2024-10-25 at 13 58 56" src="https://github.com/user-attachments/assets/8cb7b573-c180-4dbd-bf51-3dc70c73667d">
<img width="1855" alt="Screenshot 2024-10-25 at 13 58 49" src="https://github.com/user-attachments/assets/2fd608f0-a046-4299-a40b-8df041e6d656">
<img width="1855" alt="Screenshot 2024-10-25 at 13 58 32" src="https://github.com/user-attachments/assets/cf88d54c-274d-4456-81c2-936fd28d94d1">
<img width="1855" alt="Screenshot 2024-10-25 at 13 57 55" src="https://github.com/user-attachments/assets/f9448601-8915-486d-83f3-7916f2054830">
<img width="1855" alt="Screenshot_2024-10-25_at_13_58_22" src="https://github.com/user-attachments/assets/d3b58369-fa22-4559-aad1-0b8ecf5c89d6">

**Comment by gmenziesint** [comment]
Thanks for sending these over, so if I'm understanding correctly it's a replacement for IntuneCD as we already have that setup in Azure DevOps, this would replace that and provide a nicer looking GUI is that correct?

**Reply by almenscorner** [reply]
Yes, if you set up a schedule in IntuneCD Monitor, this is the one that instead is running the recurring backups/updates towards your tenants and git repo.

**Reply by gmenziesint** [reply]
Can you use IntuneCD Monitor as a viewer for the DevOps repo only without the SQL database etc? I'd like to keep the pipeline seperate and just use the monitor as a viewpoint for users.

**Reply by almenscorner** [reply]
Currently, no. DB is needed for other things as well but currently there is no way of updating the change/update log without running it from the solution itself
